src/pages/Appointment/Appointment.js

```diff
@@ -0,0 +1,214 @@
+import React from "react";
+import { View, Button, Alert, Text, Image, FlatList,TouchableOpacity  } from "react-native";
+import Background from "../../components/Background/Background";
+import {StyleSheet} from 'react-native';
+
+
+const Appointment = ({navigation} ) => {
+  const Items = [
```

Comment:
Dummy data key name is better if refer to the db schema
https://github.com/Five-Fishes/On-Demand-Car-Servicing-Backend/blob/dev/docs/schema.json